src/zh/yymanhua/src/eu/kanade/tachiyomi/extension/zh/yymanhua/YYmanhua.kt

```diff
@@ -0,0 +1,160 @@
+        .build()
+
+    // Popular Page
+
+    override fun popularMangaRequest(page: Int) = GET("$baseUrl/manga-list-p$page", headers)
+
+    override fun popularMangaSelector() = ".mh-item"
+
+    override fun popularMangaFromElement(element: Element) = SManga.create().apply {
+        title = element.select(".mh-item-detali .title a").text()
+        element.select("a:nth-child(1)").let {
+            thumbnail_url = it.select("img").attr("src")
+            url = it.attr("href")
+        }
+    }
+
+    // override fun popularMangaNextPageSelector(): String? = null
+    override fun popularMangaNextPageSelector() = ".page-pagination li:has(.active) + li a"
+
+    // Latest Page
+
+    override fun latestUpdatesRequest(page: Int) = GET("$baseUrl/manga-list-0-0-2-p$page/", headers)
+
+    override fun latestUpdatesSelector() = popularMangaSelector()
+
+    override fun latestUpdatesFromElement(element: Element) = popularMangaFromElement(element)
+
+    override fun latestUpdatesNextPageSelector() = popularMangaNextPageSelector()
+
+    // Search Page
+
+    override fun getFilterList() = buildFilterList()
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        val url = (
+            baseUrl +
+                if (query.isNotBlank()) {
+                    "/search?title=$query&page=$page"
+                } else {
+                    "/manga-list-${filters[1]}-${filters[2]}-${filters[3]}-p$page/"
+                }
+            ).toHttpUrl()
+        return GET(url, headers)
+    }
+
+    override fun searchMangaSelector() = popularMangaSelector()
+
+    override fun searchMangaFromElement(element: Element) = popularMangaFromElement(element)
+
+    override fun searchMangaNextPageSelector() = popularMangaNextPageSelector()
+
+    // Manga Detail Page
+
+    override fun mangaDetailsParse(document: Document) = SManga.create().apply {
+        description = document.select(".detail-info-content").text()
+            .replace(DESC_REGEX, "").trim()
+        val els = document.select(".detail-info-tip > span")
+        els[0].select("a").let {
+            author = it[0].text()
+            artist = it.getOrNull(1)?.text()
+        }
+        status = when (els[1].select("span")[1].text()) {
+            "连载中", "連載中" -> SManga.ONGOING
+            "已完结", "已完結" -> SManga.COMPLETED
+            else -> SManga.UNKNOWN
+        }
+        genre = els[2].select(".item").joinToString { it.text() }
+        initialized = true
+    }
+
+    // Manga Detail Page / Chapters Page (Separate)
+
+    override fun chapterListRequest(manga: SManga) = GET(baseUrl + manga.url, headers)
+
+    override fun chapterListSelector() = "#chapterlistload a"
+
+    override fun chapterFromElement(element: Element) = SChapter.create().apply {
+        url = element.attr("href")
+        name = element.text()
+        chapter_number = CHAPTER_REGEX.find(name)?.groups?.get(1)?.value?.toFloat() ?: -0F
```

Comment:
`-0F` seems wrong. I think only `-1F` and `-2F` had special meaning. Please double check.